api/src/main/java/keywhiz/api/model/SanitizedSecret.java

```diff
@@ -76,7 +76,7 @@ public static SanitizedSecret fromSecretSeriesAndContent(SecretSeriesAndContent
         series.type().orElse(null),
         series.generationOptions(),
         content.expiry(),
-        series.currentVersion().orElse(null));
+        content.id());
```

Comment:
I'm not sure I totally understand this change. I'll let someone else on the team review this line.